### PR TITLE
Add ticker to storybook

### PR DESCRIPTION
**Pull request by joelochlann**
Also

- moves it into a general components directory (`components/ticker`)
- includes CSS directly with ES6 import

In sum: makes the ticker more reusable

### CODE REVIEW

File: support-frontend/assets/components/ticker/contributionTicker.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// -----  gu-sass ----- //
+@import '~stylesheets/gu-sass/gu-sass';
```
Comment (jlieb10) [inline support-frontend/assets/components/ticker/contributionTicker.scss]: Maybe add a note to say this is bc the component is in Storybook

File: support-frontend/assets/components/ticker/contributionTicker.jsx [hunk 2]
```diff
@@ -36,7 +38,7 @@ const percentageTotalAsNegative = (total: number, goal: number) => {
 
 
 // ----- Component ----- //
-export class ContributionTicker extends Component<PropTypes, StateTypes> {
+export default class ContributionTicker extends Component<PropTypes, StateTypes> {
```
Comment (jlieb10) [inline support-frontend/assets/components/ticker/contributionTicker.jsx]: Maybe we should document work TBD in the class somewhere. Like: (1) we'd like to edit the component to get its values either from JSON or with props and (2) add some documentation about the structure of the JSON the component expects.

File: support-frontend/assets/components/ticker/contributionTicker.scss (hunk 1 above)
Comment (walaura) [reply support-frontend/assets/components/ticker/contributionTicker.scss]: Nope! this import is also picked up by `support-frontend` [as normal](https://github.com/guardian/support-frontend/wiki/CSS) we started this migration a bit ago

File: support-frontend/assets/components/ticker/contributionTicker.jsx (hunk 2 above)
Comment (joelochlann) [reply support-frontend/assets/components/ticker/contributionTicker.jsx]: I've actually documented those in this Trello card instead: https://trello.com/c/qXxukzxY/395-ticker-in-storybook-improvements-for-future-campaigns

**Pull request merged**